docker/runtime-entrypoint.sh

```diff
@@ -31,6 +31,11 @@ fi
 : "${TRACING_ENDPOINT_PORT:=3000}"
 # [rpc]
 : "${RPC_LISTEN_ADDR:=0.0.0.0}"
+if [[ "${NETWORK}" = "Mainnet" ]]; then
+: "${RPC_PORT:=8232}"
+elif [[ "${NETWORK}" = "Testnet" ]]; then
+: "${RPC_PORT:=18232}"
+fi
```

Comment:
Optional: it's fairly easy to expose a port when running the image, but it'd be nice to expose `RPC_PORT` in the Dockerfile too.

Reply:
I assume you mean the `EXPOSE` instruction. It doesn't actually do anything unless you use the `-P` (`--publish-all`) flag with `docker run`. I would personally get rid of it completely because when used with the flag, it'll map all the listed ports to random ports on the host, even though Zebra doesn't use them. Also, if we let users pick their port at build time and a potentially different port at run time, the port in `EXPOSE` might differ from the one used by Zebra.

Reply:
> I assume you mean the EXPOSE instruction.

Yep.

> It doesn't actually do anything unless you use the -P (--publish-all) flag

Well that explains a lot. Okay.